### ISSUE TITLE
Maven skip parameter for LicenseScout

### ISSUE THREAD

**Issue by pfistererm** [aposin/LicenseScout#6]
The maven configuration of the LicenseScout should provide an easy way to disable the execution of the LicenseScout is some situations.
# Feature request

## Expected Behavior - What where you expecting to happen?
 A common Maven pattern is to provide a Maven configuration parameter named 'skip' with possible values 'true' or 'false'.

So it should be possible to have a POM file like this:

```xml
<plugin>
    <groupId>org.aposin.licensescout</groupId>
    <artifactId>licensescout-maven-plugin</artifactId>
     <configuration>
         <skip>true</skip>
          ...
    </configuration>
    ...
</plugin>
```

If the parameter is given and is true, the execution of the main MOJO of the LicenseScout should be terminated. No configurations files should be read.
The default value of the parameter should be false (so that the behaviour if the parameter is not given is to do execute the licensescout).

## Current Behavior - What happens?
There is no possibility to disable the execution of the licensescout.

## Possible Solution
The parameter should be implemented in `AbstractScanMojo`.
`AbstractScanMojo.execute()` should check the value of this parameter before any other action. On value true a return should be done.
## Context
The parameter would help to manage the execution of the LicenseScout in complex build environments. Especially, it would allow to enable or disable execution of the LicenseScout in subprojects or by using a maven profile in an easy way.

## Notes and further information

---

EDITED (@d-gs): improve readability of code

**Comment by pfistererm** [comment]
In the description of the POM configuration of this request there are less than and greater than signs, that are not shown in a Web browser. Is there a way to quote them so that they are displayed correctly?

**Comment by d-gs** [comment]
For quoting code, like in this case, you should use the [Markdown](https://guides.github.com/features/mastering-markdown/) syntax
 In this case I will edit your comment and you can have a look to the edited version to see what I've changed (I am not sure how to put literal markdown syntax within comments).

**Comment by d-gs** [comment]
We decided to take this into account.
PR for this issue are highly welcome!

**Comment by MarkusMit** [comment]
**_+1_** for this feature!
It would also allow to easily deactivate the LicenceScout for faster CI builds.

**Comment by holzerma** [comment]
Please feel free to overtake! I did not yet finished the feature

**Comment by pfistererm** [comment]
Closed because implemented by #63.
